### DISCUSSION TITLE
Enterprise Help

### DISCUSSION THREAD

**Discussion by djjrock** [deepgram/community#808] [General help]
How does any one get enterprise help or support? I am already a customer and I have filled out the form for enterprise support multiple times. Does any one @team-deepgram read those emails. please help

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Is the form you are referring to one that was found in your console dashboard? I'm sorry you've been waiting; I will make sure we get you in touch with your support person.

**Reply by djjrock** [reply]
Yes, i have filled it out at least 5 times. No one ever answers or writes me back. You can find my account my name but happy to provide an email or other comms.

**Reply by SandraRodgers** [reply]
Could you send me your project ID? Feel free to email me at [sandrarodgers@deepgram.com](mailto:sandrarodgers@deepgram.com).

**Reply by djjrock** [reply]
Address not found
--
Your message wasn't delivered to sandrarodgers@deepgram.com because the address couldn't be found, or is unable to receive mail.

**Reply by djjrock** [reply]
@SandraRodgers are u messsing with me or what? This is crazy.

**Reply by SandraRodgers** [reply]
I'm so sorry. It's sandra.rodgers@deepgram.com.
